geode-benchmarks/src/main/java/org/apache/geode/benchmark/parameters/Utils.java

```diff
@@ -15,17 +15,43 @@
 
 package org.apache.geode.benchmark.parameters;
 
-import static org.apache.geode.benchmark.topology.ClientServerTopology.Roles.CLIENT;
-import static org.apache.geode.benchmark.topology.ClientServerTopology.Roles.LOCATOR;
-import static org.apache.geode.benchmark.topology.ClientServerTopology.Roles.SERVER;
+import static org.apache.geode.benchmark.Config.jvmArgs;
+import static org.apache.geode.benchmark.topology.RoleKinds.GEODE_PRODUCT;
+import static org.apache.geode.benchmark.topology.Roles.CLIENT;
+import static org.apache.geode.benchmark.topology.Roles.LOCATOR;
+import static org.apache.geode.benchmark.topology.Roles.SERVER;
 
+import java.util.Arrays;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.apache.geode.benchmark.topology.RoleKinds;
+import org.apache.geode.benchmark.topology.Roles;
 import org.apache.geode.perftest.TestConfig;
 
-public interface Utils {
+public class Utils {
+
+  static final Logger logger =
+      LoggerFactory.getLogger(Utils.class);
+
+  private Utils() {}
+
+  /**
+   * We have many settings we want to apply to JVMs that are hosting Geode. Not all JVMs
+   * host Geode. This method applies the setting to only the Geode product JVMs.
+   * @param config
+   * @param args
+   */
+  public static void configureGeodeProductJvms(final TestConfig config, final String... args) {
+    Roles.rolesFor(GEODE_PRODUCT).forEach(role -> jvmArgs(config,role,args));
+  }
```

Comment:
The new JVM role, `PROXY`, doesn't need all the Geode-specific configuration options such as GC settings. This method is used only for settings that pertain to Geode, hence the new method name.

Also, rather than hard-coding the list of pertinent roles, this method now reflects on the `Roles` enum so as that enum changes we won't have to change this method.